### PR TITLE
Move to automated dependabot merging

### PR DESCRIPTION
**Pull request by skjnldsv**
Currently this auto merge patch releases.

While I think it's fine to auto-merge minor on other apps.
I'd be more cautious on server

cc @nextcloud/javascript

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+version: 2
+updates:
+- package-ecosystem: composer
+  directory: "/"
+  schedule:
+    interval: weekly
+    day: saturday
+    time: "03:00"
+    timezone: Europe/Paris
```
Comment (ChristophWurst) [inline .github/dependabot.yml:9]: But I want `Europe/Vienna`!

**Review by ChristophWurst — APPROVED** [review]

**Review by georgehrke — CHANGES_REQUESTED** [review]
The `dependabot.yml` is missing plenty of our current config:
49244B25-F351-48D6-A810-F540FF9CC808

**Comment by skjnldsv** [comment]
Done @georgehrke !
I think it doesn't make sense to keep upgrading npm in /build for stablexx versions.
Testui integration has been removed from master now and is outdated.

**Comment by skjnldsv** [comment]
Hum, actually, do we want to keep upgrading `/build/integration` for stablexx branches?
I would only upgrade main npm to be honest

**Comment by skjnldsv** [comment]
Capture d’écran_2020-09-07_14-56-08
Damnit

**Review by georgehrke — APPROVED** [review]
Looks good to me, but i can't judge the part regarding the integration test suite.

**Comment by juliushaertl** [comment]
> Hum, actually, do we want to keep upgrading /build/integration for stablexx branches?

Yeah just security updates i'd say for those.

We can just adjust the string of the branches when backporting the dependabot config to the stable branches.

**Comment by skjnldsv** [comment]
> Yeah just security updates i'd say for those.
> 
> We can just adjust the string of the branches when backporting the dependabot config to the stable branches.

No, because dependabot doesn't check other branches than master
So we need to adjust master only.
Regarding security updates, I think they follow an independent release process. So it doesn't matter rather we have a dependabot config or not as long as it's enabled in https://github.com/nextcloud/server/network/alerts

**Comment by skjnldsv** [comment]
https://github.com/nextcloud/server/network/updates

**Pull request merged**